### PR TITLE
add catro.is-a.dev

### PR DESCRIPTION
**Pull request by catrophie**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] The file's name is all lowercased and alphanumeric. 
- [x] You have completed your website. 
- [ ] The website is reachable.  
- [x] You're not using Vercel.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.  

## Website Link/Preview

image

### CODE REVIEW

**Review by MaskDuck — CHANGES_REQUESTED** [review]

File: domains/catro.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+    "description": "My personal website :)",
+    "repo": "https://github.com/catrophie/catro.is-a.dev",
+    "owner": {
+        "username": "catrophie",
+        "email": "ycat.chayan@gmail.com",
+        "discord": "catrophie"
+    },
+    "record": {
+        "CNAME": "catrophie.github.io"
```
[the remark this answers is not in the document]
Comment (catrophie) [reply domains/catro.json:10]: it broke it self and it is refusing to fix what do i do

[the remark this answers is not in the document]
Comment (catrophie) [reply domains/catro.json:10]: https://catrophie.github.io/catro.is-a.dev/

[the remark this answers is not in the document]
Comment (Yunexiz) [reply domains/catro.json:10]: > 404 website

github wants to use CNAME directly to the users github.io (which you cant reach). the actual ones are hosted with /repo-name

**Comment by orxngc** [comment]
Closing as stale.

**Pull request closed without merging**